src/index.js

```diff
@@ -22,15 +25,19 @@ exports.codes = Object.freeze(codes)
 /**
  * Convert the given multihash to a hex encoded string.
  *
- * @param {Buffer} hash
+ * @param {Uint8Array} hash
  * @returns {string}
  */
 exports.toHexString = function toHexString (hash) {
-  if (!Buffer.isBuffer(hash)) {
-    throw new Error('must be passed a buffer')
+  if (!(hash instanceof Uint8Array)) {
+    throw new Error('must be passed an Uint8Array')
```

Comment:
Nitpick (not critical, more fyi). It's "*a* Uint8array" as it depends on the pronunciation, "Uint" starts with "ju…".

Reply:
I always thought if it was based on first latter being a vowel 🤯